bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/obs/ObsListView.scala

```diff
@@ -687,6 +687,19 @@ class ObsListView(shellAdvisor:ShellAdvisor, band:Band, queueLookup: Target => U
       }
     }
 
+    // Edit the blueprint
+    def fixBlueprint(b: BlueprintBase) {
```

Comment:
Yes... I tested it and it opens the blueprint editor for the given blueprint and saves the changes as appropriate!